packages/element/src/react-platform.native.js

```diff
@@ -1,50 +0,0 @@
-/**
- * External dependencies
- */
-import { AppRegistry } from 'react-native';
-import { omit } from 'lodash';
-
-/**
- * WordPress dependencies
- */
-import { applyFilters, doAction } from '@wordpress/hooks';
-
-/**
- * Internal dependencies
- */
-import { Component, cloneElement } from './react';
-
-const render = ( element, id ) => {
```

Comment:
We no longer require this file as the App registration is now done in `react-native-editor`.